ext/standard/crc32.c

```diff
@@ -68,6 +68,8 @@ static inline int has_crc32_insn(void) {
 #  if!defined(__clang__)
 #   pragma GCC push_options
 #   pragma GCC target ("+nothing+crc")
+#  elif defined(__APPLE__)
+#   pragma clang attribute push(__attribute__((target("crc"))), apply_to=function)
```

Comment:
This misses a pragma pop down below

Reply:
Added. Although, I think `attribute pop` would be handled with `#else`.

Reply:
> Added. Although, I think `attribute pop` would be handled with `#else`.

Right, but at least now it's symmetric :)